apps/ca.c

```diff
@@ -586,7 +585,7 @@ int ca_main(int argc, char **argv)
             && (certfile = lookup_conf(conf, section, ENV_CERTIFICATE)) == NULL)
             goto end;
 
-        x509 = load_cert(certfile, certformat, "CA certificate");
+        x509 = load_cert(certfile, certformat, NULL, "CA certificate");
```

Comment:
It would probably be better to leave this function signature alone and add another one that handles this variant. Internally the old function just calls the new one with the extra NULL parameter.

Reply:
All right, done.
On this occasion I also made clear in the documentation files that the cert input file format specifications are no longer needed and are ignored (but are retained for backward compatibility).